src/event-handlers/court-emails.js

```diff
@@ -193,20 +194,35 @@ const createLambdaHandler = (createWeb3, createContract) => {
       });
     }
 
-    const web3 = await _web3();
-    const chainId = await web3.eth.getChainId();
-    const sendgrid = await _sendgrid();
-    const { PRIVATE_KEY } = await getEnvVars(['PRIVATE_KEY']);
+    const web3 = await createWeb3()
+    const chainId = await web3.eth.getChainId()
+    const sendgrid = await _sendgrid()
+    const { PRIVATE_KEY } = await getEnvVars(['PRIVATE_KEY'])
     const lambdaAccount = web3.eth.accounts.privateKeyToAccount(
       PRIVATE_KEY.replace(/^\s+|\s+$/g, '')
     );
 
     const handlers = createEventHandlers(chainId);
-    const notifications = await handlers[event.event](
-      web3,
-      createContract(web3),
-      event
-    );
+    let notifications = []
+    try {
+      notifications = await handlers[event.event](
+        web3,
+        createContract(web3),
+        event
+      );
+    } catch (err) {
+      const { event: eventName, ...payload } = event
+      console.info(`Error processing event ${event.event} with payload:`, payload)
```

Comment:
Why not `console.error` here?

Reply:
I probably copy-pasted the line and forgot to change the method.